chart/values.yaml

```diff
@@ -131,7 +131,14 @@ siem_env:
   arkime_network_index_pattern: "arkime_sessions3-*"
   # Default time field used by for sessions in Arkime viewer
   arkime_network_index_time_field: "firstPacket"
-
+  # Allows user to add a default_pipeline to malcolm_template or malcolm_beat_template
```

Comment:
the non-beats stuff here may not be used yet at the moment, due to zeek/suricata being split out from each other (see the logstash section above), but the capability is here now to specify these values for the network logs as a whole if desired. If they're left blank, then they're just left out of the template completely. See also cisagov/Malcolm#657 as this is something I haven't quite figured out how to be able to completely remove your overrides yet.